lib/begin.go

```diff
@@ -179,6 +190,9 @@ func (a *ACBuild) beginFromImage(start string, insecure bool) error {
 
 	err = reg.Fetch(app.Name, labels, 0, false)
 	if err != nil {
+		if _, ok := err.(*url.Error); ok {
+			return fmt.Errorf("unknown host when fetching image, local file paths must start with '/' or '.'")
```

Comment:
but are we sure it's an unknown host error? This feels like it might be catching too generously